pkg/get/get_test.go

```diff
@@ -3312,38 +3312,56 @@ func Test_DownloadKrew(t *testing.T) {
 		{
 			os:      "darwin",
 			arch:    arch64bit,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-darwin_amd64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-darwin_amd64.tar.gz`,
 		},
 		{
 			os:      "darwin",
 			arch:    archARM64,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-darwin_arm64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-darwin_arm64.tar.gz`,
+		},
+		{
+			os:      "darwin",
+			arch:    archDarwinARM64,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-darwin_arm64.tar.gz`,
 		},
 		{
 			os:      "linux",
 			arch:    archARM64,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-linux_arm64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-linux_arm64.tar.gz`,
 		},
 		{
 			os:      "linux",
 			arch:    arch64bit,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-linux_amd64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-linux_amd64.tar.gz`,
 		},
 		{
 			os:      "linux",
 			arch:    archARM7,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-linux_arm.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-linux_arm.tar.gz`,
 		},
 		{
 			os:      "ming",
 			arch:    arch64bit,
-			version: "v0.4.2",
-			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.2/krew-windows_amd64.tar.gz`,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-windows_amd64.tar.gz`,
+		},
+		{
+			os:      "ming",
+			arch:    archARM64,
+			version: "v0.4.3",
+			url:     `https://github.com/kubernetes-sigs/krew/releases/download/v0.4.3/krew-darwin_arm64.tar.gz`,
+		},
+		{
+			os:      "ming",
```

Comment:
What's this entry for? Windows + MacOS 64-bit?

Reply:
Yes, I meant to comment here: https://github.com/alexellis/arkade/pull/837/files#diff-6e1468cd6d67c262166efacb61ddf1e9dbb404be9ca1acd3ce0c2c2abc95a741R1031

To the effect that i didn't understand the darwin entry under ming but added the additional arch entry for consistency